### ISSUE TITLE
Usersnap Feedback - Hyperlink issues (with accented letters/ can't unlink)

### ISSUE THREAD

**Issue by davidascher** [participedia/usersnaps#152]
**Sender**: scottowenfletcher@gmail.com
**Comment**: Hyperlink ends at accented vowel but the actual URL for the page includes it (try copy-and-pasting it)
[Open #165 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-ed971123/165)

<a href='https://usersnappublic.s3.amazonaws.com/2018-04-16/16-51/91c49cfe-6720-4bbd-b5d2-f853557f32ed.png'>![Usersnap Feedback - Hyperlink ends at accented vowel but the actual URL for the page includes (...)](https://usersnappublic.s3.amazonaws.com/2018-04-16/16-51/aa7214cc-b727-4f37-a17e-b4409d623a70.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2018-04-16/16-51/91c49cfe-6720-4bbd-b5d2-f853557f32ed.png'>Download original image</a>

**Browser**: Chrome 65 (Windows 10)
**Referer**: [https://participedia.xyz/case/4225](https://participedia.xyz/case/4225)
**Screen size**: 1280 x 720 **Browser size**: 1422 x 600

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by jesicarson** [comment]
Original issue:

![screen shot 2018-05-23 at 11 57 48 am](https://user-images.githubusercontent.com/13560530/40445564-54d44f72-5e81-11e8-8e30-8e9a524cdfc2.png)


Tried to edit the narrative but came across two problems:

1. Highlighting the url text and clicking the "unlink/break link" button did not break the existing link.

2. Copy and pasting this url 
![screen shot 2018-05-23 at 11 59 36 am](https://user-images.githubusercontent.com/13560530/40445440-ec7024e2-5e80-11e8-9729-8b7d2cfa5143.png)
 (including accented characters) into the narrative body produced the following: https://www.opendemocracy.net/kristinn-m%C3%A1r-%C3%A1rs%C3%A6lsson/real-democracy-in-iceland (with other characters representing the accented characters). While this link still brings you to the same page, it doesn't look good)

3. Because the link wouldn't break (item 1), I had to re-paste the url and re-link it. It now works, but also doesn't display correctly. In addition to the accented characters being replaced, the underline of the url doesn't extend along the entire url: 
![screen shot 2018-05-23 at 12 00 17 pm](https://user-images.githubusercontent.com/13560530/40445577-5dda7c9a-5e81-11e8-8ac8-4b9b26bb9b37.png)
